optuna/study/_multi_objective.py

```diff
@@ -119,12 +124,15 @@ def _calculate_nondomination_rank(
     objective_values: np.ndarray,
     *,
     n_below: int | None = None,
+    base_rank: int = 0,
 ) -> tuple[np.ndarray, int]:
-    # Calculate the domination matrix.
-    # The resulting matrix `domination_matrix` is a boolean matrix where
+    # Normalize n_below.
+    n_below = n_below or len(objective_values)
+    n_below = min(n_below, len(objective_values))
```

Comment:
if `n_below` is non-positive, we can already return the results using `base_rank`.

Reply:
Thanks for your comment. I added an eraly return black.